integration/tests/01-missing-permissions.spec.js

```diff
@@ -0,0 +1,36 @@
+const { test, expect } = require("@playwright/test");
+const { KubeappsLogin } = require("./utils/kubeapps-login");
+const utils = require("./utils/util-functions");
+
+test("Regular user fails to deploy an application due to missing permissions", async ({ page }) => {
+  test.setTimeout(60000);
+  
+  // Log in
+  const k = new KubeappsLogin(page);
+  await k.doLogin("kubeapps-user@example.com", "password", process.env.VIEW_TOKEN);
+
+  // Select package to deploy
+  await page.click('a.nav-link:has-text("Catalog")');
+  await page.click('a .card-title:has-text("apache")');
+  await page.click('cds-button:has-text("Deploy") >> nth=0');
+
+  // Deploy package
+  const releaseNameLocator = page.locator("#releaseName");
+  await releaseNameLocator.waitFor();
+  await expect(releaseNameLocator).toHaveText("");
+  await releaseNameLocator.type(utils.getRandomName("test-01-release"));
+  await page.locator('cds-button:has-text("Deploy")').click();
+
+  const errorLocator = page.locator(".alert-items .alert-text");
+  await expect(errorLocator).toHaveCount(1);
+  await page.waitForTimeout(5000);
+
+  // For some reason, UI is showing different error messages randomly
```

Comment:
Ah - I was wondering why you weren't testing for the presence of a particular error above. Keen to understand what is random here. What are some examples of what you see for the error message in the console below? Approx. what frequency etc?

Reply:
We are getting [both messages shown here](https://github.com/kubeapps/kubeapps/issues/4102).
Basically one is "unable to get secrets" and the other is "unable to create secrets". Most of the times one of them is thrown (don't remember which one from the top of my head), but every once in a while, the other is appearing instead. This is not related with testing, it is an issue that we should investigate.